File: Java/Vetores/Negativos.java
package Java.Vetores;

import java.util.Locale;
import java.util.Scanner;


public class Negativos {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int N, i;

        System.out.print("Quantos numeros voce vai digitar? ");
        N = sc.nextInt();

        int[] vet = new int[N];

        for (i = 0; i < N; i++) {
            System.out.print("Digite um numero: ");
            vet[i] = sc.nextInt();
        }

        System.out.println();
        System.out.println("NUMEROS NEGATIVOS:");

        for (i = 0; i < N; i++) {
            if (vet[i] < 0) {
                System.out.println(vet[i]);
            }
        }

        sc.close();

    }
}

File: C++/Vetores/Negativos.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;

int main()
{

    int N, i;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> N;

    int vet[N];

    for (i = 0; i < N; i++)
    {
        cout << "Digite um numero: ";
        cin >> vet[i];
    }

    for (i = 0; i < N; i++)
    {
        if (vet[i] < 0)
        {
            cout << vet[i] << endl;
        }
    }

    return 0;
}
